doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -92,9 +92,10 @@ To mount a subtree of the CephFS root, append the path to the device string::
     mount -t ceph :/subvolume/dir1/dir2 /mnt/mycephfs -o name=fs
 
 If you have more than one file system on your Ceph cluster, you can mount the
-non-default FS on your local FS as follows::
+non-default FS on your local FS as follows(Or the deprecated ``mds_namespace=<fs-name>``
```

Comment:
I'd shorten this:

"non-default FS as follows:"

...and leave off the part about mds_namespace. This document is really about how we recommend mounting things, so we don't really need to cover legacy options.

Reply:
Okay, will fix it.